src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -252,7 +254,7 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 		this._sameTaskTerminals = Object.create(null);
 		this._onDidStateChange = new Emitter();
 		this._taskSystemInfoResolver = taskSystemInfoResolver;
-		this._register(this._terminalStatusManager = new TaskTerminalStatus(taskService));
+		this._register(this._terminalStatusManager = new TaskTerminalStatus(taskService, audioCueService));
```

Comment:
Create this via instantiation service?

Reply:
tried to and had to revert

Reply:
https://github.com/microsoft/vscode/pull/160828/commits/4ddfd3b7ffe2348c5dc126248fc1eedb58fc4daa